<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server_Info_Form</class>
 <widget class="QDialog" name="Server_Info_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>442</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>442</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lab_ip">
         <property name="text">
          <string>服务端IP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_ip">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lab_port">
         <property name="text">
          <string>服务端端口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_port">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lab_dataType">
         <property name="text">
          <string>数据类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_dataType">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lab_publicAddr">
         <property name="text">
          <string>公共地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_publicAddr">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="lab_describe">
         <property name="text">
          <string>描述</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MyTextEdit" name="textEdit_server_des">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>70</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>220</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Save">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">MyTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
